test/accessibility/services/a11y/a11y.ts

```diff
@@ -33,86 +33,84 @@ export const normalizeResult = (report: any) => {
   return report.result as false | AxeReport;
 };
 
-export function A11yProvider({ getService }: FtrProviderContext) {
-  const browser = getService('browser');
-  const Wd = getService('__webdriver__');
-
-  /**
-   * Accessibility testing service using the Axe (https://www.deque.com/axe/)
-   * toolset to validate a11y rules similar to ESLint. In order to test against
-   * the rules we must load up the UI and feed a full HTML snapshot into Axe.
-   */
-  return new (class Accessibility {
-    public async testAppSnapshot(options: TestOptions = {}) {
-      const context = this.getAxeContext(true, options.excludeTestSubj);
-      const report = await this.captureAxeReport(context);
-      await this.testAxeReport(report);
-    }
+/**
+ * Accessibility testing service using the Axe (https://www.deque.com/axe/)
+ * toolset to validate a11y rules similar to ESLint. In order to test against
+ * the rules we must load up the UI and feed a full HTML snapshot into Axe.
+ */
+export class AccessibilityService extends FtrService {
+  private readonly browser = this.ctx.getService('browser');
+  private readonly Wd = this.ctx.getService('__webdriver__');
```

Comment:
even though the generated `d.ts` file defines them as `any`:
```typescript
export declare class AccessibilityService extends FtrService {
    private readonly browser;
    private readonly Wd;
    testAppSnapshot(options?: TestOptions): Promise<void>;
    testGlobalSnapshot(options?: TestOptions): Promise<void>;
    private getAxeContext;
    private assertValidAxeReport;
    private captureAxeReport;
}
```
Both WebStorm and VSCode  infer the final type correctly:
![2021-06-03_12-31-38](https://user-images.githubusercontent.com/3198181/120623475-0da8c180-c460-11eb-9900-3e9bc06bf4f5.png)
![2021-06-03_12-30-07](https://user-images.githubusercontent.com/3198181/120623489-0f728500-c460-11eb-9fea-860c49bf2c57.png)

Reply:
I think it's because they're `private`, no other reason for `assertValidAxeReport` to be typed the same.